guardian/newsletters-nx
# Added new instance running the read only api

## What does this change?

This adds a new EC2 instance running as a separate app configured not to serve the UI and to provide read-only access to the API.

This will be used to serve external clients (Braze, Identity & Frontend). A future PR will add a Fastly deployment to cache these responses.

<img width="1169" alt="Screenshot 2023-05-11 at 14 12 02" src="https://github.com/guardian/newsletters-nx/assets/3277259/68595a3e-e9d9-47d6-8cf4-ec3df1337aeb">


## How to test

Deploy and ensure the existing app is running and that the  [read-only API](https://readonly-newsletters.code.dev-gutools.co.uk/) is available and that is is not possible to write to it




## Have we considered potential risks?

This offers unauthenticated access to the Newsletters API. This is the same as the existing newsletters api and we are not exposing sensitive data
